styles/breakpoints.css

```diff
@@ -1,7 +1,7 @@
 @custom-media --lido-breakpoint-xs-down (max-width: 374px);
-@custom-media --lido-breakpoint-sm-down (max-width: 599px);
+@custom-media --ui-breakpoint-mobile (max-width: 599px);
 @custom-media --lido-breakpoint-md-down (max-width: 899px);
-@custom-media --lido-breakpoint-ml-down (max-width: 1199px);
+@custom-media --ui-breakpoint-mid-desctop (max-width: 1199px);
```

Comment:
What is the difference between --ui-breakpoint and --lido-breakpoint? If there is a difference, I suggest putting it in a comment